tests/test_scheduler.rs

```diff
@@ -397,19 +396,24 @@ async fn assert_rcc_files_permissions(
 }
 
 async fn assert_rcc_configuration(rcc_config: &RCCConfig) -> AnyhowResult<()> {
-    let mut rcc_config_diag_command = Command::new(&rcc_config.binary_path);
-    rcc_config_diag_command
-        .arg("configuration")
-        .arg("diagnostics");
-    let stdout = String::from_utf8(rcc_config_diag_command.output().await?.stdout)?;
-    assert!(stdout.contains("telemetry-enabled                     ...  \"false\""));
-    assert!(stdout.contains("holotree-shared                       ...  \"true\""));
-    assert!(stdout.contains("holotree-global-shared                ...  \"true\""));
+    let diagnostics = read_configuration_diagnostics(&rcc_config.binary_path)?;
+    assert_eq!(
+        diagnostics
+            .details
+            .get("telemetry-enabled")
+            .unwrap()
+            .as_str(),
+        "false"
+    );
```

Comment:
In the last commit, we should explicitly test that the shared holotree is disabled if possbile

Reply:
Done